bindings/wasm/scripts/build

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
 
-wasm-pack build --no-pack --target nodejs
+# get target as argument from cli, defaults to nodejs if no argument is supplied
+TARGET=${1:-nodejs}
+FEATURE="nodejs"
+
+if [ "$TARGET" = "web" ]; then
+    FEATURE="web"
+fi
+
+wasm-pack build --no-pack --target $TARGET --no-default-features --features $FEATURE
```

Comment:
could we add `wasm-pack` as an `npm` dev dependency and call `npx wasm-pack` here?